prototype2/client/src/engine/control/event_distribution.rs

```diff
@@ -5,18 +5,21 @@ use engine;
 
 use glutin::Event::KeyboardInput;
 use glutin::Event::MouseMoved;
+use world::ExitFlag;
```

Comment:
Personal pref, but for bool newtypes like ExitFlag can we impl Copy and not ref it?